### DISCUSSION TITLE
How to suppress "scrapli will try to escalate privilege"?

### DISCUSSION THREAD

**Discussion by denisbondar** [carlmontanari/scrapli#320] [Q&A] [answered]
Hi!
The console gets messages like this all the time. And people often come to me with this question, thinking it's something scary.
The thing is that some devices may not contain the Enable password and I don't know it in advance.

What is the right way to suppress these messages so that my colleagues stop asking me the same question?

```
*************************** Authentication Warning! ****************************
scrapli will try to escalate privilege without entering a password but may fail.
Set an 'auth_secondary' password if your device requires a password to increase privilege, otherwise ignore this message.
********************************************************************************
```

**Comment by carlmontanari** [comment, marked as answer]
ahah a good question :D this has come up before and we do have a solution!

there is a `from scrapli.settings import Settings; Settings.SUPPRESS_USER_WARNINGS = true` should do the trick!
